### PR TITLE
[DPDK] Skip build dependencies for package manager installation

### PR DESCRIPTION
**Pull request by mcgov**
Manual installation of ninja and meson is buggy across versions, skip this process for package manager installation since it's not needed anyway.

### CODE REVIEW

File: microsoft/testsuites/dpdk/dpdktestpmd.py [hunk 1]
```diff
@@ -576,6 +576,8 @@ def _install_dependencies(self) -> None:
                     list(self._ubuntu_packages_1804),
                     extra_args=self._debian_backports_args,
                 )
+                if self._dpdk_source == "package_manager":
```
Comment (squirrelsc) [inline microsoft/testsuites/dpdk/dpdktestpmd.py]: 1. How about define a constant for "package_manager"?
2. The skipped logic is long. How about add some private method like `_install_ubuntu_build_dependencies`? So the logic will be clearer. Either install by package manager or by built. The logic is like,
```python
if self._dpdk_source != PACKAGE_MANAGER:
    self._install_ubuntu_build_dependencies()
```

Comment (mcgov) [reply microsoft/testsuites/dpdk/dpdktestpmd.py]: Excellent! I will break it up

Comment (mcgov) [reply microsoft/testsuites/dpdk/dpdktestpmd.py]: This is so much cleaner I should have done this a long time ago.

Comment (squirrelsc) [reply microsoft/testsuites/dpdk/dpdktestpmd.py]: I learned that not too long ago. The function is not only to reuse code, it provides meaningful logic as well.

**Review by squirrelsc — DISMISSED** [review]

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**